src/asset/search.js

```diff
@@ -61,3 +65,45 @@ function setupToggleSearch() {
 }
 
 window.addEventListener('DOMContentLoaded', setupToggleSearch)
+
+//Theme
+function toggleTheme() {
+  const theme = localStorage.getItem('theme')
+  if (theme === 'dark')
+    setLightTheme()
+  else if (theme === 'light')
+    setDarkTheme()
+}
+
+function setDarkTheme() {
+  var dark_css = document.querySelector("link[href='a11y-dark.css']")
+  var light_css = document.querySelector("link[href='a11y-light.css']")
+  dark_css.setAttribute("media", "")
+  light_css.setAttribute("media", "false")
+  localStorage.setItem('theme', 'dark')
+  document.querySelector('html').classList.remove("light-theme")
+  document.querySelector('html').classList.add("dark-theme")
+}
+
+function setLightTheme() {
+  var dark_css = document.querySelector("link[href='a11y-dark.css']")
+  var light_css = document.querySelector("link[href='a11y-light.css']")
+  dark_css.setAttribute("media", "false")
+  light_css.setAttribute("media", "")
+  localStorage.setItem('theme', 'light')
+  document.querySelector('html').classList.remove("dark-theme")
+  document.querySelector('html').classList.add("light-theme")
+}
+
+function setupToggleTheme() {
+  Array.from(document.querySelectorAll('.theme-toggle')).forEach(el => {
+    el.addEventListener('click', toggleTheme)
+  })
+}
+
+window.addEventListener('DOMContentLoaded', setupToggleTheme)
+
+if (localStorage.getItem('theme') === 'dark')
+  setDarkTheme();
+if (localStorage.getItem('theme') === 'light')
+  setLightTheme();
```

Comment:
No theme will be setup of local storage is empty.

Probably want to put setLightTheme in `else`.